de.schmidhuberj.tubefeeder.json

```diff
@@ -0,0 +1,63 @@
+{
+    "app-id" : "de.schmidhuberj.tubefeeder",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "40",
+    "sdk" : "org.gnome.Sdk",
+    "sdk-extensions" : [
+        "org.freedesktop.Sdk.Extension.rust-stable"
+    ],
+    "add-extensions": { "org.freedesktop.Platform.ffmpeg-full": {
+        "directory": "lib/ffmpeg",
+        "version": "21.08beta",
+        "add-ld-path": "."
+      }
+    },
+    "command": "tubefeeder",
+    "finish-args": [
+        "--socket=fallback-x11",
+        "--socket=wayland",
+	"--socket=pulseaudio",
+        "--device=dri",
+        "--share=network",
+	"--filesystem=xdg-data/tubefeeder:create",
+	"--filesystem=xdg-cache/tubefeeder:create",
+	"--filesystem=xdg-config/mpv:ro"
```

Comment:
since you build pipewire-jack by way of MPV you need the following:
```
    "--filesystem=xdg-run/pipewire-0",
    "--system-talk-name=org.freedesktop.RealtimeKit1"
```